samples/net/sockets/coap_service/README.rst

```diff
@@ -0,0 +1,57 @@
+.. zephyr:code-sample:: coap-service
+   :name: CoAP service
+   :relevant-api: coap udp
+
+   Use the CoAP server subsystem that exposes CoAP resources.
+
+Overview
+********
+
+This sample is a simple CoAP server showing how to expose a simple resource.
```

Comment:
So the question is - shall this new sample be considered a replacement for the old `coap_server` sample? I don't think there's a need to have two samples exposing similar functionality.

Reply:
Replaced the existing sample using the new services.